src/clojure_lsp/crawler.clj

```diff
@@ -296,18 +296,22 @@
                   (str (get-in (run-kondo! source-paths kondo-user-config) [:summary :duration]) "ms"))
         (async/<!! crawler-output-chan)))))
 
-(defn find-project-settings [project-root]
+(defn find-raw-project-settings [project-root]
   (let [config-path (Paths/get ".lsp" (into-array ["config.edn"]))]
     (loop [dir (uri->path project-root)]
       (let [full-config-path (.resolve dir config-path)
             file (.toFile full-config-path)
             parent-dir (.getParent dir)]
         (cond
           (.exists file)
-          (edn/read-string {:readers {'re re-pattern}} (slurp file))
```

Comment:
it was needed to declare this function that just return the file string otherwise, the returned config edn to the user is not human readable.